src/utils/byte_parsing.ts

```diff
@@ -26,6 +26,16 @@ type TypedArray = Int8Array |
                   Float32Array |
                   Float64Array;
 
+function UTF16StrToBytes(string: string): Uint16Array {
```

Comment:
Missing JSDoc and explanation of what it does

Reply:
my bad